packages/belos/src/BelosPseudoBlockGmresSolMgr.hpp

```diff
@@ -758,32 +779,52 @@ setParameters (const Teuchos::RCP<Teuchos::ParameterList>& params)
 
     if (tempOrthoType != orthoType_) {
       orthoType_ = tempOrthoType;
+
       params_->set("Orthogonalization", orthoType_);
+
+      if (haveOrthoSublist) {
+        params_->set("OrthoManager", *orthoParams);
```

Comment:
The list that is returned by getCurrentParameters() needs to be readable if it is given to setParameters().  The use of "OrthoManager" as the sublist name doesn't seem to be consistent with how the parameter list is processed by the solver manager, as it is looking for a sublist having the name of the orthogonalization type.